### PR TITLE
Update openoffice to 4.1.5

### PR DESCRIPTION
**Pull request by gtmgianni**
After making all changes to the cask:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by commitay** [comment]
Why add all the languages?

**Comment by commitay** [comment]
@gtmgianni Please remove the additional languages unless you actually use them.

**Comment by gtmgianni** [comment]
@commitay There was no particular reason for adding all of them, other than for the sake of completeness. I’ll change it back to the previous language stanzas that were included, as I am not sure which languages people will need to use.

**Comment by vitorgalvao** [comment]
> I am not sure which languages people will need to use.

A good rule of thumb is “you need it, you add it”.

File: Casks/openoffice.rb [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
     'fr'
   end
 
   language 'gl' do
-    sha256 'b27988df0ceea64e2a1bde8e148dcd0dd9f322979320778027c63270d2967a0b'
+    sha256 'edfdd62c2f3ada01cb39f73e529ac0665b21b43b649b4a2699f69e6ebcbad26c'
     'gl'
   end
 
-  language 'pt-BR' do
```
Comment (commitay) [inline Casks/openoffice.rb]: Please change this back, the order of the language blocks is important. https://github.com/caskroom/homebrew-cask/issues/42755

**Pull request merged**